pkg/controller/operators/catalog/operator.go

```diff
@@ -138,7 +141,7 @@ func NewOperator(ctx context.Context, kubeconfigPath string, clock utilclock.Clo
 		client:                   crClient,
 		lister:                   lister,
 		namespace:                operatorNamespace,
-		resolver:                 resolver.NewOperatorsV1alpha1Resolver(lister, crClient, opClient.KubernetesInterface()),
+		resolver:                 resolver.NewInstrumentedResolver(res, success, failure),
```

Comment:
I had to google `Instrumented: equip (something) with measuring instruments.`

Reply:
Ins't that clever? :p